data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -550,9 +546,36 @@ def prepare_aar_data(aar_path, current_year: int):
 
     aar["Is PFI"] = aar["PFI School"].map(lambda x: x == "PFI School")
 
+    # TODO: any processing required from `pre_process_trust_aar()` below?
```

Comment:
`prepare_trust_aar_data()` below can be dropped entirely but:

- is it worth removing the Trust-level processing in this function at the moment?
- should entries without a Co. No. be dropped?

Reply:
- Trust-level processing removed (these are added later during Academy processing.)
- `prepare_trust_aar_data()` removed.